deploy/gk-deploy

```diff
@@ -239,7 +239,7 @@ while [[ $# -ge 1 ]]; do
   shift
 done
 
-if [[ ! -f ${TOPOLOGY} ]]; then
+if [[ ${ABORT} -ne 1 ]] && [[ ! -f ${TOPOLOGY} ]]; then
```

Comment:
No need for an extra check, just move the entire topology check block to https://github.com/gluster/gluster-kubernetes/blob/master/deploy/gk-deploy#L303 just after the main abort check.